src/release.js

```diff
@@ -233,17 +233,19 @@ export default function taoExtensionReleaseFactory(params = {}) {
                 releaseBranch,
                 data.version,
                 data.lastVersion,
-                subjectType
+                subjectType,
             );
-
             if (pullRequest && pullRequest.state === 'open') {
                 data.pr = {
                     url: pullRequest.html_url,
                     apiUrl: pullRequest.url,
                     number: pullRequest.number,
-                    id: pullRequest.id
+                    id: pullRequest.id,
+                    full_name: pullRequest.head.repo.full_name,
                 };
-
+                const labels = ['releases'];
+                await githubClient.addLabel(data.pr.full_name, data.pr.number, labels);
+                // await githubClient.addLabel(`oat-sa/tao-extension-release-new`,3,'releases');
```

Comment:
Remove the dead code please (check other modified files too).

Reply:
updated